Features/Standard/ContentTypeIcon.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Content Type icons
+
+    @javascript
+    Scenario: I can see the generic Content Type icon attached to the Content item name
+        Given a Content Type "custom_content_type" exists
```

Comment:
Having the identifier in there doesn't seem very relevant, since it is not used in the rest of the scenario.

What's particular with the content type used in this scenario ?